avocado/utils/network/interfaces.py

```diff
@@ -571,3 +572,22 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def check_packet_loss(self, peer_ip, count, option=None, flood=False):
+        """
+        Function checks for packet loss during ping.
+        Return False for 0% packet loss and True if packet loss occurs.
+        """
+        cmd = f'ping -I {self.name} {peer_ip} -c {count}'
+        if flood is True:
+            cmd = f'{cmd} -f'
+        elif option is not None:
+            cmd = f'{cmd} {option}'
```

Comment:
IIUC, option is only used when not in flood mode. This should be properly documented at the docstring.